script/1_0_2current_en_script/Common/00_02_hidden.txt

```diff
@@ -20,7 +20,7 @@ The entire clearing is painted in the deepest red.
 
 <Page13>
 みんなをそうしたように ボクもバラバラにしたいみたい.
-He looks like he wants to tear me apart just like everyone else.
+He looks like he wants to tear me apart, like he did to everyone else.
```

Comment:
Hmm, this reads a bit off to me still. Can't think of something better this moment though.